crates/cxx-qt-lib/src/core/qpointf.rs

```diff
@@ -73,6 +91,48 @@ impl fmt::Display for QPointF {
     }
 }
 
+impl std::ops::Add for QPointF {
+    type Output = Self;
+    fn add(self, other: Self) -> Self {
+        ffi::qpointf_plus(&self, &other)
+    }
+}
+
+impl std::ops::Sub for QPointF {
+    type Output = Self;
+    fn sub(self, other: Self) -> Self {
+        ffi::qpointf_minus(&self, &other)
+    }
+}
+
+impl std::ops::Mul<f32> for QPointF {
+    type Output = Self;
+    fn mul(self, rhs: f32) -> Self {
+        ffi::qpointf_mul_f32(rhs, &self)
+    }
+}
+
+impl std::ops::Mul<f64> for QPointF {
```

Comment:
I'll try building this in a Wasm environment when i get a moment on the right machine ...

Reply:
> Which is more correct here and will work across platforms ?

None of them satisfies both concerns.
The one working across platforms is `f32`, as it will implicitly cast to `f64` when qreal is double, so `f32` seems favorable.
But it is only favorable when you don't need the precision of `f64` on the Rust side. Not really sure either what is the best option here.

Reply:
Looking at https://github.com/KDAB/cxx-qt/commit/e334c2907f776f8e9a729d7a0cc9e3b55b39b8a0 again, it seems a `double` / `qreal` works in Wasm (it does have support for 64bit floating point numbers just `int` is 32bit etc). So using `f64` might be OK here as it is `qreal` (aka `double`) in both Qt 5 and Qt 6. And should work with Wasm ...

Reply:
(I think the fun comes from when it is `qreal` in Qt 5 and `float` in Qt 6, which is not the case here luckily)

Reply:
So assuming that is the case, it's probably best to go with `f64` in these places and lets see if it breaks :-)

Reply:
Right, however i don't think we test / support armhf / armel / ARMv6/v7 at all (aka any ARM 32bit or even x86), so there are likely other issues if that is the case.

We know that arm64 / ARMv8-A / AArch64 works and that Wasm works with some patches though.